admin/views/dashboard-widget.php

```diff
@@ -37,6 +37,9 @@ class="wpseo-glance <?php echo esc_attr( $statistic['class'] ); ?>">
 		</tr>
 	<?php endforeach; ?>
 </table>
+<p><?php printf( _n( 'You have %d post that is already well optimized, well done!', 'You have %d posts that are already well optimized, well done!', $good_posts, 'wordpress-seo' ), $good_posts ); ?></p>
```

Comment:
Even with a default value of `0`, these strings won't make sense saying "well done".